base-theme/layouts/partials/featured_pages.html

```diff
@@ -0,0 +1,7 @@
+{{- $pages := .context.Site.RegularPages -}}
+{{- $result := slice -}}
+{{- $featuredPagesUid := .featuredPagesUid -}}
+{{ range $featuredPagesUid}}
+  {{ $result = $result | append (where $pages "Params.uid" "==" .) }}
+{{ end }}
+{{ return $result }}
```

Comment:
**Suggestion:** A more generic name like `filtered_pages.html` might be appropriate for this partial.

Reply:
@ChristopherChudzicki Should we just default to all promos in case there are no featured promos ?

Reply:
My inclination would be to leave the empty array as the default: less chance of publishing something unintentionally. But I think both are reasonable.

Reply:
whatever is easier / more reliable. In all likelihood, this will be updated two or three times a year.

Reply:
Whatever is easy / robust is fine. The way this is used I expect there will always be a list and it will be updated a couple times a year.